### PR TITLE
Implemented -f option for the clean command

### LINKED ISSUE #1398: micromamba clean missing `-f` flag
We are missing the `-f, --force-pkgs-dirs` option in micromamba clean, which is used by repo2docker.

### PR DESCRIPTION
**Pull request by JohanMabille**
Fixes #1398

### CODE REVIEW

**Comment by JohanMabille** [comment]
The current implementation is missing a confirmatino question.

File: libmamba/src/api/clean.cpp [hunk 1]
```diff
@@ -292,6 +293,14 @@ namespace mamba
             }
         }
 
+        if (clean_force_pkgs_dirs)
+        {
+            for (auto& cache: ctx.pkgs_dirs)
```
Comment (wolfv) [inline libmamba/src/api/clean.cpp:298]: maybe better to use `            for (auto* pkg_cache : caches.writable_caches())`?

File: micromamba/src/clean.cpp [hunk 2]
```diff
@@ -70,6 +76,16 @@ set_clean_command(CLI::App* subcom)
             options = options | MAMBA_CLEAN_LOCKS;
         if (config.at("clean_trash").compute().value<bool>())
             options = options | MAMBA_CLEAN_TRASH;
+        if (config.at("clean_force_pkgs_dirs").compute().value<bool>())
+        {
+            std::cout << "Remove all contents from the package caches ([y]/n)?" << std::endl;
```
Comment (wolfv) [inline micromamba/src/clean.cpp:81]: Can you use `Console::prompt(...)`?

Comment (wolfv) [reply micromamba/src/clean.cpp:81]: And mabye we should move thsi to the actual place where things are removed? And print what package dirs are going to be erased.

**Comment by gsakkis** [comment]
Currently the -y/--yes flag is not respected with -f:
```
micromamba clean  -f -y
Remove all contents from the package caches?: [y/n] 
```

**Comment by wolfv** [comment]
@gsakkis please file a new issue :)

**Comment by gsakkis** [comment]
[Done!](https://github.com/mamba-org/mamba/issues/1562)

**Pull request merged**